app/helpers/episodes_helper.rb

```diff
@@ -119,6 +119,15 @@ def episode_medium_options
     Episode.mediums.keys.map { |k| [I18n.t("helpers.label.episode.mediums.#{k}"), k] }
   end
 
+  def episode_media_updated_at(episode)
```

Comment:
i put this in a helper but i'm wondering if it should just be a method in the model itself? i don't know if it would be a useful method outside of the UI.

Reply:
A helper makes sense for now - we'll see if it's useful elsewhere!

The `def episode_media_status` above does something similar to this - concating the episode media and uncut, to derive a single "status" for all of the media.  Could be worth consolidating that `all_media` logic?

Also saves a nil check, since you'd just get an empty `all_media == []` array.  And then you can do something like `all_media.maximum(:updated_at)`.  (Maximum is a [Rails method](https://apidock.com/rails/Enumerable/maximum) added to enumerables)